src/nvim/if_cscope.c

```diff
@@ -1509,7 +1532,8 @@ static char *cs_parse_results(int cnumber, char *buf, int bufsize, char **contex
  */
 static void cs_file_results(FILE *f, int *nummatches_a)
 {
-  int i, j;
+  size_t i;
+  int j;
```

Comment:
> However sometimes, for example when a loop counter is reused in several loops which are not nested, I would argue that declaring them at the start is more readable.

I suppose it increases readability from the perspective of the lines being shorter, but IMO it's more obvious that a for loop counter isn't being used anywhere else if it's declared inline.